m/mesa/stone.yaml

```diff
@@ -103,15 +109,15 @@ setup       : |
         -Dgles1=disabled \
         -Dglvnd=enabled \
         -Dllvm=enabled \
+        -Dmesa-clc=system \
         -Dosmesa=true \
         -Dzstd=enabled \
         -Dplatforms=x11,wayland \
         -Dshared-llvm=enabled \
         -Dvideo-codecs=all \
         -Dmicrosoft-clc=disabled \
-        -Dintel-clc=system \
-        -Dvulkan-drivers=amd,intel,intel_hasvk,swrast,virtio$EXTRA_VK_DRIVERS \
-        -Dvulkan-layers=device-select,overlay,screenshot
+        -Dvulkan-drivers=amd,gfxstream,intel,intel_hasvk,swrast,virtio$EXTRA_VK_DRIVERS \
+        -Dvulkan-layers=device-select,intel-nullhw,overlay,screenshot,vram-report-limit
```

Comment:
`intel-nullhw` is really only for driver devs and is unnecessary for us.

Reply:
Removed.